auth/iam-token-manager-v1.ts

```diff
@@ -0,0 +1,158 @@
+/**
+ * Copyright 2015 IBM Corp. All Rights Reserved.
```

Comment:
Need to check copyright date on this file.

Reply:
Yes, that's certainly wrong. The IAM token manager was originally written in 2018 but this specific file in this repo was added in 2019. So - should it be `2018, 2019` or `2019`?